.changeset/few-fans-bathe.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fix: `no-invalid-double-slash-comments` reported `endIndex`
```

Comment:
[suggest] How about adjusting it a bit? Because `endIndex` is an internally related thing.

```suggestion
Fixed: `no-invalid-double-slash-comments` reported end positions
```

Reply:
```suggestion
Fixed: `no-invalid-double-slash-comments` end positions